webapp/svelte.config.js

```diff
@@ -1,7 +1,12 @@
-import { vitePreprocess } from "@sveltejs/vite-plugin-svelte";
+import adapter from "@sveltejs/adapter-node";
+import { svelte, vitePreprocess } from "@sveltejs/vite-plugin-svelte";
 
 export default {
   // Consult https://svelte.dev/docs#compile-time-svelte-preprocess
   // for more information about preprocessors
   preprocess: [vitePreprocess({})],
+
+  kit: {
+    adapter: adapter()
```

Comment:
what is this adapter for?

Reply:
changes adapter from vite to node look at new import.

Reply:
why that change? What is it needed for?